### ISSUE TITLE
自定义的图标 如何使用figma插件呢 还有vscode插件  如果能提供这样的功能就是最棒的

### ISSUE THREAD

**Issue by fengchuanBIG** [iconify/api#23]
自定义的图标 如何使用figma插件呢 还有vscode插件  如果能提供这样的功能就是最棒的

**Comment by cyberalien** [comment]
Are you asking how to use Figma plugin with custom API?

I'm not sure what the question is related to VSCode.

**Comment by fengchuanBIG** [comment]
Yes, I have deployed it privately, how to use my own api on the figma plugin.
  It would be nice if the vscode plugin could also support the API for privatized deployment

**Comment by cyberalien** [comment]
Currently it cannot be added, but that might be possible in new version.

As for VSCode plugin, I do not have any VSCode plugins, so you need to contact author of plugin you are using.
